addons/website_knowledge/views/knowledge_templates_frontend.xml

```diff
@@ -20,6 +20,7 @@
                 <ul class="o_tree o_tree_workspace m-0 px-0 py-2">
                     <t t-call="knowledge.articles_template">
                         <t t-set="articles" t-value="published_articles"/>
+                        <t t-set="portal_readonly_mode" t-value="True"/>
```

Comment:
Same comment, should already be set to True here:
https://github.com/odoo/odoo/blob/master/addons/knowledge/controllers/main.py#L88